File: ejercicios parte 1/2)estructuras condicionales/13division/division.java
import java.util.Scanner;

public class division {
    public static void main(String[] args) {
        try (Scanner scan = new Scanner(System.in)) {
            int dividendo, divisor;
            System.out.print("Ingrese el dividendo: ");
            dividendo = scan.nextInt(); 
            System.out.print("Ingrese el divisor: ");
            divisor = scan.nextInt();
            if(divisor==0){
                System.out.println("No se puede dividir por 0");
                return;
            }
            if(dividendo%divisor == 0){
                System.out.println("La división es exacta");          
            }else{
                System.out.println("La división NO es exacta");
            }
            System.out.println("Cociente: " + dividendo/divisor);
            System.out.println("Residuo: " + dividendo%divisor);  
        }
    }    
}

File: ejercicios parte 1/2)estructuras condicionales/13division/division.cpp
#include <iostream>
using namespace std;
int main(){
    int n1, n2;
    cout<<"Dividendo: ";
    cin>>n1;
    cout<<"Divisor: ";
    cin>>n2;
    if(n2==0){
        cout<<"No se puede dividir entre 0"<<endl;
        return 0;
    }
    if(n1%n2==0){
        cout<<"La división es exacta"<<endl;
    }else{
        cout<<"La división no es exacta"<<endl;
    }
    cout<<"Cociente: "<<n1/n2<<endl;
    cout<<"Resto: "<<n1%n2<<endl;
    return 0;
}